test/jobs/benchmark_regression_job_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class BenchmarkRegressionJobTest < ActiveJob::TestCase
+  test "#perform benchmark_regression" do
+    benchmark_result_type = create(:benchmark_result_type)
+    benchmark_type = create(:benchmark_type)
+    commit = create(:commit)
+    initiator = commit
+    benchmark_run = create(:commit_benchmark_run,
+      benchmark_result_type: benchmark_result_type,
+      benchmark_type: benchmark_type,
+      initiator: commit
+    )
+    category = "sometime"
+    BenchmarkRegressionJob.new.perform(benchmark_run, initiator, benchmark_type, benchmark_result_type, category)
+  end
+end
```

Comment:
Can you please read through http://guides.rubyonrails.org/testing.html.

I am not following how your test is actually testing that the job you added works